Src/WitsmlExplorer.Frontend/components/Modals/DeleteEmptyMnemonicsModal.tsx

```diff
@@ -0,0 +1,84 @@
+import Wellbore from "../../models/wellbore";
+import Well from "../../models/well";
+import JobService, { JobType } from "../../services/jobService";
+import { TextField } from "@material-ui/core";
+import OperationType from "../../contexts/operationType";
+import { useContext, useState } from "react";
+import { HideModalAction } from "../../contexts/operationStateReducer";
+import { DeleteEmptyMnemonicsJob } from "../../models/jobs/deleteEmptyMnemonicsJob";
+import ModalDialog from "./ModalDialog";
+import OperationContext from "../../contexts/operationContext";
+import { DateTimeField } from "./DateTimeField";
+import styled from "styled-components";
+
+export interface DeleteEmptyMnemonicsModalProps {
+  wells?: Well[];
+  wellbores?: Wellbore[];
+  dispatchOperation: (action: HideModalAction) => void;
+}
+
+const DeleteEmptyMnemonicsModal = (props: DeleteEmptyMnemonicsModalProps): React.ReactElement => {
+  const { wells, wellbores, dispatchOperation } = props;
+  const {
+    operationState: { timeZone }
+  } = useContext(OperationContext);
+  const [nullDepthValue, setNullDepthValue] = useState<number>(-999.25);
+  const [nullTimeValue, setNullTimeValue] = useState<string>("1900-01-01T00:00:00.000+00:00");
```

Comment:
I was inprecise in the issue description, the default nullTimeValue should be "1900-01-01T00:00:00.000Z".